### DISCUSSION TITLE
Limit to process data

### DISCUSSION THREAD

**Discussion by 5011-5011** [nextgenhealthcare/connect#5645] [Q&A] [answered]
Hello!
One question, does mirth have a limit to process data? in open source version.
I'm trying to extract data but the channel doesn't go up, it stays stopped. I have other data extraction channels but this one is giving me problems.

reference image attached
![image](https://user-images.githubusercontent.com/98954172/215569427-6b137158-0611-4500-8257-ecba28fec3d6.png)

![image](https://user-images.githubusercontent.com/98954172/215569673-cb236689-ac50-4d38-9e15-35ba95be8abe.png)

**Comment by pacmano1** [comment, marked as answer]
No limit.  You have a problem in your channel - it cannot initialize the db connection so you need to track down why that is.

**Reply by 5011-5011** [reply]
@pacmano1  Thanks for the reply! I will check
